src/libmongoc/src/mongoc/mongoc-client.c

```diff
@@ -1406,7 +1406,8 @@ mongoc_client_get_collection (mongoc_client_t *client,
                                   collection,
                                   client->read_prefs,
                                   client->read_concern,
-                                  client->write_concern);
+                                  client->write_concern,
+				  NULL);
```

Comment:
Why not simply have `_mongoc_collection_new` take the `timeout_ms` here instead of the database? That's what we do for the other options, e.g., `write_concern`.

Reply:
I've changed this to just hold the timeout value, but we only take it if it's set on the database, otherwise it remains unset in the collection as well.